components/presence/presence.stories.js

```diff
@@ -0,0 +1,70 @@
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import PresenceMdx from './presence.mdx';
+import Presence from './presence.vue';
+import { PRESENCE_STATES_LIST } from './presence_constants';
+import PresenceDefaultTemplate from './presence_default.story.vue';
+import PresenceVariantsTemplate from './presence_variants.story.vue';
+
+export const argTypesData = {
+  presence: {
+    table: {
+      type: {
+        summary: 'string',
+      },
+    },
+    control: {
+      type: 'select',
+      options: [...PRESENCE_STATES_LIST],
+    },
+  },
+  theme: {
+    table: {
+      type: {
+        summary: 'string',
+      },
+    },
+    control: {
+      type: 'select',
+      options: ['none', 'default'],
+    },
+  },
+};
+
+// Story Collection
+export default {
+  title: 'Components/Presence',
+  component: Presence,
+  argTypes: argTypesData,
+  excludeStories: /.*Data$/,
+  parameters: {
+    controls: {
+      sort: 'requiredFirst',
+    },
+    docs: {
+      page: PresenceMdx,
+    },
+    options: {
+      showPanel: true,
+    },
+  },
+};
+
+// Templates
+const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  PresenceDefaultTemplate,
+);
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  PresenceVariantsTemplate,
+);
```

Comment:
This is not displaying anything in storybook right now, need to remove argTypes from these statements for vue 3. Should be like this:

`const DefaultTemplate = (args) => createTemplateFromVueFile(args, PresenceDefaultTemplate);`

Reply:
My bad, I updated this here: https://github.com/dialpad/dialtone-vue/pull/569/commits/967b6631f546013491eb088c5995ee78f62ce5c3.

However, I'm still struggling to figure out why I'm seeing this error now:
![image](https://user-images.githubusercontent.com/88498639/199257091-524fc8bc-2ca2-4913-ae53-e77a4cf4bfa3.png)